src/thorin/transform/closure_conversion.cpp

```diff
@@ -51,7 +51,7 @@ class ClosureConversion {
         for (auto pair : converted) {
             if (pair.second != pair.first) {
                 pair.first->destroy_body();
-                pair.first->make_internal();
+                world_.make_internal(pair.first);
```

Comment:
Do we really have to do this considering `pair.first` is dead at that point ?

Reply:
If `p.first` is external, it is live by definition.